test-support/ember-sinon-qunit/test.js

```diff
@@ -20,11 +20,21 @@ sinon.config = {
 };
 
 export default function test(testName, callback) {
+  // Enable adding to the context (i.e. `this` during the `test` function)
+  test.context = {};
```

Comment:
Instead of creating `test.context`, why don't we just merge `this` with `context` on line 35?